From c3a12b318be2ad98b5fc89ff4a1b8061d69d0097 Mon Sep 17 00:00:00 2001
From: kumarkir <kiran2.kumar@intel.corp-partner.google.com>
Date: Wed, 26 Feb 2020 22:13:51 -0800
Subject: [PATCH 2/2] [DO NOT MERGE] CHROMIUM: media: ov8856: Modify PCLK
 PERIOD

This patch modifies ov8856 sensor's PCLK PERIOD to make
it work on Intel's JSL+ platform. Do not merge this patch.

BUG=b:149514656

TEST=Sensor tested on JSL+ platform.

Change-Id: Ibbe4550dc2db9f2a4290b302620973d0d401d394
Signed-off-by: kumarkir <kiran2.kumar@intel.corp-partner.google.com>
---
 drivers/media/i2c/ov8856.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/ov8856.c b/drivers/media/i2c/ov8856.c
index 8655842af275..d8b18cf7bd75 100644
--- a/drivers/media/i2c/ov8856.c
+++ b/drivers/media/i2c/ov8856.c
@@ -264,7 +264,7 @@ static const struct ov8856_reg mode_3280x2464_regs[] = {
 	{0x4816, 0x53},
 	{0x481b, 0x58},
 	{0x481f, 0x27},
-	{0x4837, 0x16},
+	{0x4837, 0x0e},
 	{0x483c, 0x0f},
 	{0x484b, 0x05},
 	{0x5000, 0x57},
-- 
2.25.0.265.gbab2e86ba0-goog

